tests/playwright/utilities/terminal.js

```diff
@@ -0,0 +1,36 @@
+import { execSync } from 'node:child_process';
+import path from 'path';
+
+class Terminal {
+	/*
+     * Activate or Deactivate experiments
+     * @param action activate | deactivate
+     * @param experiments comma separated list of experiments. e.g. container,nested-elements,'grid container'
+     */
+	async experiments( action, experiments ) {
+		const wpPath = await this.getWordpressInstallPath();
+
+		const wpEnv = `npx wp-env run cli "bash -c 'wp elementor experiments ${ action } ${ experiments }'"`;
+		const local = `bash -c "wp elementor experiments ${ action } ${ experiments } --path=${ wpPath }"`;
+
+		const result = await this.executeTerminal( process.env.CI ? wpEnv : local );
+		console.log( result );
+	}
+	async executeTerminal( cmd ) {
+		try {
+			const stdout = execSync( cmd );
```

Comment:
Not sure if we should somehow sanitise the `cmd`?

Reply:
@louiswol94 I've restricted allowable cmd's to the `wp`  

This allows things like:

```
wp elementor experiments activate container
wp elementor experiments default
wp plugin activate woocommerce 

```

And would fail with other terminal commands.
What do you think?